app-modules/audit/database/migrations/2024_07_22_092622_data_migration_agent_name.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2024, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use Laravel\Pennant\Feature;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Database\Migrations\Migration;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        if (Feature::active('change-agent-name')) {
+            try {
+                DB::beginTransaction();
+
+                $audits = DB::table('audits')
+                    ->join('users', 'audits.change_agent_id', '=', 'users.id')
+                    ->select('audits.id', 'users.name')
+                    ->get();
+
+                foreach ($audits as $audit) {
+                    DB::table('audits')
+                        ->where('id', $audit->id)
+                        ->update(['change_agent_name' => $audit->name]);
+                }
```

Comment:
Due to the size of the audits table we should `chunkById` here instead of `get`. https://laravel.com/docs/10.x/queries#chunking-results

We may crash memory with a `get`

A good chunk size may be something like 100.